ui/pages/settings/settings-tab/settings-tab.component.js

```diff
@@ -142,6 +145,43 @@ export default class SettingsTab extends PureComponent {
     );
   }
 
+  renderTheme() {
+    const { t } = this.context;
+    const { theme, setTheme } = this.props;
+
+    const themesOptions = [
+      {
+        name: t('defaultTheme'),
+        value: THEME_TYPE.DEFAULT,
+      },
+      {
+        name: t('darkTheme'),
+        value: THEME_TYPE.DARK,
+      },
+    ];
```

Comment:
I think the branch needs a rebase because the `osTheme` seems to be missing in both the experimental and the General tab.

```suggestion
      {
        name: t('osTheme'),
        value: THEME_TYPE.OS,
      },
    ];
```

Reply:
We are also missing the `light` theme from the options. 

In the latest develop branch, `themesOptions` is:

```

    const themesOptions = [
      {
        name: t('lightTheme'),
        value: THEME_TYPE.LIGHT,
      },
      {
        name: t('darkTheme'),
        value: THEME_TYPE.DARK,
      },
      {
        name: t('osTheme'),
        value: THEME_TYPE.OS,
      },
    ];
  ```